src/app/hm-admin/hm-admin.component.ts

```diff
@@ -1,23 +1,46 @@
 import {Component, Input, OnInit} from '@angular/core';
 import {HTTP_PROVIDERS} from '@angular/http';
+import {Route} from '@angular/router';
+import {MdButton} from '@angular2-material/button';
+import {MdIcon, MdIconRegistry} from '@angular2-material/icon';
 import {ConfigService} from './services/config/Config.service';
 import {CoreDefinitionService} from './services/core/CoreDefinition.service';
 import {SchemaService} from './services/schema/schema.service';
+import {HeaderComponent} from './header/index'
+import {ContentComponent, SidenavService} from './content/index'
+import {HomeComponent} from './+home';
+import { ListComponent } from './+list';
```

Comment:
are we spaces nazis in this project? If so, well, there is two spaces here that we should drop